<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="topLeftX_label">
            <property name="text">
             <string>Top Left X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="tolLeftX_sb">
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="topLeftY_label">
            <property name="text">
             <string>Top Left Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="topLeftY_sb">
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="width_label">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="width_sb">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="height_label">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="height_sb">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="arrow_rb">
          <property name="text">
           <string>Arrow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ellipse_rb">
          <property name="text">
           <string>Ellipse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="square_rb">
          <property name="text">
           <string>Square</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="angle_sb">
          <property name="minimum">
           <double>-500.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="arrowButton">
        <property name="text">
         <string>Add arrow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ellipseButton">
        <property name="text">
         <string>Add ellipse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="squareButton">
        <property name="text">
         <string>Add square</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="regthSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
